Source/WebCore/css/html.css

```diff
@@ -847,6 +856,31 @@ input:is([type="checkbox"], [type="radio"]):checked:disabled {
 #endif
 }
 
+input[type=checkbox]:not([switch]):checked::checkmark {
+    content: '\2713' / '';
+    height: 100%;
+    width: 100%;
+    display: inline-block;
+    background-color: currentColor;
+}
+
+input[type=radio]:checked::checkmark {
+    background-color: currentColor;
+    display: inline-block;
+    border-radius: inherit;
+    margin: 10%;
+    height: 80%;
+    width: 80%;
+}
+
+input:is([type=checkbox]:not([switch]), [type=radio]):not(:checked)::checkmark {
+    display: none;
+}
+
+input:is([type=checkbox]:not([switch]), [type=radio]):checked:disabled::checkmark {
+    background-color: color-mix(in srgb, currentColor 30%, transparent);
```

Comment:
Doesn't match [the spec](https://drafts.csswg.org/css-forms/#basic-appearance-stylesheet). Presumably the spec needs updating here, are there any outstanding decisions that need to be edited in or do we need to file an issue?